static/fetch_lib.sh

```diff
@@ -3,33 +3,27 @@
 true
 # see https://github.com/koalaman/shellcheck/wiki/Directive
 
-# Functions
-msg_box() {
-    [ -n "$2" ] && local SUBTITLE=" - $2"
-    whiptail --title "$TITLE$SUBTITLE" --msgbox "$1" "$WT_HEIGHT" "$WT_WIDTH"
-}
-
-
 IRed='\e[0;91m'         # Red
 IGreen='\e[0;92m'       # Green
 ICyan='\e[0;96m'        # Cyan
 Color_Off='\e[0m'       # Text Reset
 print_text_in_color() {
-        printf "%b%s%b\n" "$1" "$2" "$Color_Off"
+    printf "%b%s%b\n" "$1" "$2" "$Color_Off"
 }
 
 mkdir -p /var/scripts
 if ! [ -f /var/scripts/lib.sh ]
 then
     if ! curl -sfL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh -o /var/scripts/lib.sh
     then
-        msg_box "You don't seem to have an internet connection and the local lib isn't available. Hence you cannot run this script."
+        print_text_in_color "$IRed" "You don't seem to have an internet connection and the local lib isn't available. Hence you cannot run this script."
         exit 1
     fi
-else
+elif test "$(find /var/scripts/lib.sh -mmin +30)"
+then
     print_text_in_color "$ICyan" "Updating lib..."
     curl -sfL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh -o /var/scripts/lib.sh
 fi
 
 # shellcheck source=lib.sh
-source /var/scripts/lib.sh &>/dev/null
+source /var/scripts/lib.sh
```

Comment:
this is really not necessary!
But if you do it here, I would suggest that you also remove it from anywhere else

Reply:
That's the plan :)